src/ai/backend/common/events.py

```diff
@@ -221,7 +222,7 @@ def deserialize(cls, value: tuple):
 class ImagePullStartedEvent(AbstractEvent):
     name = "image_pull_started"
 
-    image: str = attrs.field()
+    image: ImageRef = attrs.field()
```

Comment:
Even if the type of event field is changed, will there be no compatibility issues?

Reply:
The manager and agent versions must match and be at least 25.3.1 to avoid compatibility issues.

If the ImagePullStartedEvent does not provide the architecture information of the image, the manager will not be able to identify the corresponding ImageRow. Therefore, the ImageRef information should be sent, or at least the architecture information should be included. (To record an `ImageAuditLog`)

Reply:
It seems safer to create a new field for use.